content/_index.md

```diff
@@ -178,11 +216,31 @@ g3Entrance:
       </div>
     </div>
   </div>
+  <div class="g3-inner-wrapper g3-flex-content g3-space__padding-lg-bottom">
+    <div class="g3-col__50">
+      <div class="g3-space__margin-sm-left g3-space__margin-sm-right">
+        <h3>{{< param "g3Feature.f5.title" >}}</h3>
+        <p class="g3-space__margin-sm-top">
+          {{< param "g3Feature.f5.detail" >}}
+        </p>
+      </div>
+    </div>
+    <div class="g3-col__50">
+      <div class="g3-space__margin-sm-left g3-space__margin-sm-right">
+        <h3>{{< param "g3Feature.f6.title" >}}</h3>
+        <p class="g3-space__margin-sm-top">
+          {{< param "g3Feature.f6.detail" >}}
+        </p>
+      </div>
+    </div>
+  </div>
 </section>
 
-<h2 class="g3-text__center g3-space__margin-sm-bottom">
-  {{< param "g3Roles.title" >}}
-</h2>
+<section>
+  <h2 class="g3-text__center g3-space__margin-sm-bottom">
+    {{< param "g3Roles.title" >}}
```

Comment:
shoudl this one be in its own section? imo it looks weird this title is in the upper white block, rather than in the gray block below

Reply:
I agree, it was like this before but I will move this one to the grey box